jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/validation/EntityValidator.java

```diff
@@ -41,20 +44,31 @@ public void validateAndThrowIfErrorsExist(Object objectToValidate) {
     }
   }
 
-  private static String createResourceConstraintViolationMessage(
-      Set<ConstraintViolation<Object>> constraintViolations) {
-
-    return constraintViolations.stream()
-        .collect(groupingBy(ConstraintViolation::getPropertyPath))
-        .entrySet().stream()
-        .map(entry -> {
-          var propertyErrors = entry.getValue().stream()
-              .map(ConstraintViolation::getMessage)
-              .collect(Collectors.joining(", "));
-
-          return String.format("%s has the following error(s): %s",
-              entry.getKey().toString(), propertyErrors);
-        })
-        .collect(Collectors.joining(". "));
+  private static Object[] createResourceConstraintViolationMessage(
```

Comment:
Probs better to avoid passing data around in arrays, and use a List instead. 
Could we change this to a List and use that type all the way through to the ApiResponseExceptionHandler?

Reply:
Made this change now :) I've also renamed the method, not sure if it's better, let me know if you'd prefer something else

Reply:
thanks, yep that's great